ctk/common/src/main/java/org/codice/compliance/utils/sign/SimpleSign.java

```diff
@@ -56,21 +58,20 @@
 import org.opensaml.xmlsec.signature.support.SignatureConstants;
 import org.opensaml.xmlsec.signature.support.SignatureValidator;
 import org.opensaml.xmlsec.signature.support.provider.ApacheSantuarioSignatureValidationProviderImpl;
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
 
 public class SimpleSign {
 
-  private static final Logger LOGGER = LoggerFactory.getLogger(SimpleSign.class);
-
   private final SystemCrypto crypto;
 
-  private static final Map<String, String> URI_ALG_MAP = new HashMap<>();
-
-  static {
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#dsa-sha1", "SHA1withDSA");
-    URI_ALG_MAP.put("http://www.w3.org/2000/09/xmldsig#rsa-sha1", "SHA1withRSA");
-  }
+  private static final ImmutableMap<String, String> URI_ALG_MAP =
+      ImmutableMap.<String, String>builder()
+          .put("http://www.w3.org/2001/04/xmldsig-more#rsa-md5", "md5WithRSA")
+          .put(WSS4JConstants.RSA, "sha1WithRSA")
+          .put(WSS4JConstants.RSA_SHA256, "sha256WithRSA")
+          .put("http://www.w3.org/2001/04/xmldsig-more#rsa-sha384", "sha384WithRSA")
+          .put("http://www.w3.org/2001/04/xmldsig-more#rsa-sha512", "sha512WithRSA")
+          .put(WSS4JConstants.DSA, "sha1WithDSA")
+          .build();
 
   public SimpleSign() throws IOException {
```

Comment:
❓ Wasn't this class much closer - identical(?) - to the DDF version previously? If it was and how now diverged, we might want to document the ways in which it is written and operates that are different from DDF.